detox/runners/jest-circus/reporters/DetoxReporter.js

```diff
@@ -1,13 +1,15 @@
-const DetoxStreamlineJestReporter = require('./DetoxStreamlineJestReporter');
+const { VerboseReporter: JestVerboseReporter } = require('@jest/reporters'); // eslint-disable-line node/no-extraneous-require
+
 const FailingTestsReporter = require('./FailingTestsReporter');
 
-class DetoxReporter extends DetoxStreamlineJestReporter {
+class DetoxReporter extends JestVerboseReporter {
```

Comment:
OK, I looked into jest's new VerboseReporter and it seems like it is indeed now equivalent to DetoxStreamlineJestReporter ✅ 
+2 points to for being great visionaries